### DISCUSSION TITLE
redis deployment is meshed but no traffic displayed in the dashboard

### DISCUSSION THREAD

**Discussion by zhoudaxia233** [linkerd/linkerd2#10333] [Help]
I have set up a Redis deployment, configured the annotation for opaque port on 6379 and make sure it is meshed. While testing the Redis manually, everything seems to be working fine. However, when I checked the dashboard, I could not see Redis in the generated topology or in the live calls.

**Comment by kleimkuhler** [comment]
Can you see Service topology and live calls for other Services in the dashboard? What is the full output of `linkerd check`? Are there errors or warnings in the logs for linkerd-viz deployments?

**Reply by zhoudaxia233** [reply]
Yes, I can see topology and live calls for other services. All checks are passed when running `linkerd check`.

**Comment by jeremychase** [comment]
@zhoudaxia233 With Redis(`TCP`) traffic we really don't expect `viz` to work, but you should still be able to find metrics in Grafana.
